### PR TITLE
increase max length of `addOnTests` log-file names

### PR DESCRIPTION
**Pull request by missirol**
#### PR description:

This PR suggests to increase slightly the maximum length of the names of log files created by the `addOnTests`.

The goal would be to avoid having the same log-file name for two different `addOnTests`, as that might apparently confuse the `cmsbot`.

https://github.com/cms-sw/cmssw/pull/36408#issuecomment-988888689 incorrectly reports a failure for `hlt_mc_Fake2`, while the failure actually occurred for `hlt_data_Fake2`. From a first (non-expert) look, this seems to come from [here](https://github.com/cms-sw/cms-bot/blob/9c756b35f970eb001038cb70512f3d0334e50e29/report-pull-request-results.py#L145), where the logic might fail if the log file of two different tests has the same basename (which is the case for the 2nd test in `hlt_mc_Fake2` and `hlt_data_Fake2`, given the limit of 150 characters). The cheap workaround is to make log-file names unique by making them slightly longer. Of course, it might be better to improve the "file-finding" logic in cms-bot, but that is left to experts.

In other words, maybe this is not the right fix, and I'm opening the PR mainly to bring this to the attention of experts.

Would require https://github.com/cms-sw/cms-bot/pull/1675.

Merely technical. No changes expected.

#### PR validation:

Checked that the log-file names of the `addOnTests` would now be unique.

#### If this PR is a backport, please specify the original PR and why you need to backport that PR:

N/A

### CODE REVIEW

**Comment by missirol** [comment]
Hi @makortel , @smuzaffar ,

I see that cms-sw/cms-bot#1675 has been merged, but this PR has been quiet.

Just for my understanding, do you already have a plan in mind for this PR?

**Comment by missirol** [comment]
I was wondering if this PR could still be useful, or should be closed.

**Comment by missirol** [comment]
I'll take the silence in this PR as an invitation to close it. :)

**Comment by mmusich** [comment]
I think this PR was neglected because it was submitted close to seasons' break? 
What about re-open it again?

**Comment by missirol** [comment]
Turns out that `160` is not enough, the following two files from the tests have the same basename.

https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-0aa471/28868/addOnTests/hlt_mc_HIon/cmsRun__cvmfs_cms-ib.cern.ch_sw_x86_64_nweek-02758_el8_amd64_gcc10_cms_cmssw-patch_CMSSW_12_6_X_2022-11-07-2300_src_HLTrigger_Configuration_test_OnLine_HLT_HIon.log

https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-0aa471/28868/addOnTests/hlt_data_HIon/cmsRun__cvmfs_cms-ib.cern.ch_sw_x86_64_nweek-02758_el8_amd64_gcc10_cms_cmssw-patch_CMSSW_12_6_X_2022-11-07-2300_src_HLTrigger_Configuration_test_OnLine_HLT_HIon.log

`170` could be sufficient, but all this suggests that a better solution is probably needed (as written back then in the description).

**Comment by missirol** [comment]
https://github.com/cms-sw/cmssw/pull/40018 tries to provide an alternative to this PR.

**Comment by missirol** [comment]
Re-closing this PR. https://github.com/cms-sw/cms-bot/pull/1879 undid https://github.com/cms-sw/cms-bot/pull/1675, so this PR is not an option anymore (and it was never really a solution). Let's see if #40018 converges.
